soaks/common/configs/http_gen_datadog_source.yaml

```diff
@@ -3,6 +3,7 @@ prometheus_addr: "0.0.0.0:9090"
 
 targets:
   vector:
+    seed: [2, 3, 5, 7, 11, 13, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127, 131, 137]
```

Comment:
Curious what's that for?

Reply:
It's the seed for randomness in the lading tool. We previously used `rand::thread_rng()` which meant that every run was allowed to vary in payload, a problem if we accidentally generate an "easy" payload for one run and a "hard" one for another.